edu.cuny.hunter.mylyngit.core/src/edu/cuny/hunter/mylyngit/core/analysis/MylynGitPredictionProvider.java

```diff
@@ -109,10 +118,21 @@ public void setJavaProjects(IJavaProject[] javaProjects) {
 
 	/**
 	 * Bump DOI when there is a method change.
+	 * 
+	 * @param scaling
+	 * @param DEFAULT_SCALING_FACTOR
 	 */
-	public void bumpDOI(IMethod method) {
+	public void bumpDOI(IMethod method, InteractionContextScaling scaling, float DEFAULT_SCALING_FACTOR) {
+
+		if (this.enclosingMethods.contains(method)) {
+			// the scaling factor for enclosing method
+			scaling.set(Kind.EDIT, (float) 5.6);
```

Comment:
This scaling factor should be a calculation that use the default value. It should not be a constant.